src/settings_translation_file.cpp

```diff
@@ -213,7 +213,7 @@ fake_function() {
 	gettext("Fall bobbing");
 	gettext("Multiplier for fall bobbing.\nFor example: 0 for no view bobbing; 1.0 for normal; 2.0 for double.");
 	gettext("3D mode");
-	gettext("3D support.\nCurrently supported:\n-    none: no 3d output.\n-    anaglyph: cyan/magenta color 3d.\n-    interlaced: odd/even line based polarisation screen support.\n-    topbottom: split screen top/bottom.\n-    sidebyside: split screen side by side.");
```

Comment:
do not edit this file. This file's content is automatically generated. Rather, edit the file `builtin/settingtypes.txt`.